src/views/EditFolder/EditFolder.tsx

```diff
@@ -88,6 +95,7 @@ export class EditFolder extends React.Component<Props> {
                     </button>
                 </nav>
                 <form>
+                    <div>{this.props.folderFail}</div>
```

Comment:
After using destructuring like const { folderFail } = this.props you don't need to use this.props any more. Just use {folderFail} here.